src/components/campaigns/InlineDonation.tsx

```diff
@@ -120,18 +130,19 @@ export default function InlineDonation({ campaign }: Props) {
   const { asPath } = useRouter()
   const [status, copyUrl] = useCopyToClipboard(baseUrl + asPath, 1000)
   const active = status === 'copied' ? 'inherit' : 'primary'
+  const [page, setPage] = useState<number>(0)
   const target = campaign.targetAmount
   const summary = campaign.summary.find(() => true)
   const reached = summary?.reachedAmount ?? 0
   const donors = summary?.donors ?? 0
   const {
-    data: { items: donations } = { items: [] },
+    data: { items: donations, total: all_rows } = { items: [] },
     error: donationHistoryError,
     isLoading: isDonationHistoryLoading,
-  } = useCampaignDonationHistory(campaign.id)
+  } = useCampaignDonationHistory(campaign.id, page, 5)
   const { mobile } = useMobile()
   const [isOpen, setIsOpen] = useState(false)
-
+  const rowCount = page * 5 + donations.length
```

Comment:
it would be good if that `5` is a const because if you change this to a different number here or on line 142, the code will no longer work. It should be obvious that these two need to have the same value